app/src/main/java/org/wikipedia/views/ImageTitleDescriptionView.kt

```diff
@@ -7,18 +7,32 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.LinearLayout
 import androidx.annotation.DrawableRes
+import androidx.annotation.NonNull
 import androidx.appcompat.content.res.AppCompatResources
 import androidx.core.content.ContextCompat
 import androidx.core.widget.ImageViewCompat
+import com.skydoves.balloon.showAlignBottom
 import kotlinx.android.synthetic.main.view_image_title_description.view.*
 import org.wikipedia.R
 import org.wikipedia.util.DimenUtil
+import org.wikipedia.util.FeedbackUtil
 
 internal class ImageTitleDescriptionView constructor(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs) {
+    var tooltipText: String = ""
 
     init {
         View.inflate(context, R.layout.view_image_title_description, this)
         layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
+        setOnLongClickListener {
+            if (!tooltipText.isEmpty()) {
+                description.showAlignBottom(FeedbackUtil.showTooltip(context, tooltipText), 0, 16)
```

Comment:
The alignment of the arrow might look even better if we anchor it to the view itself (`this`) instead of the `description` view.

Reply:
It centers that to the whole view @dbrant so Robin wanted it to be centered to the description.

Reply:
OK, Robin wants it centered on the description _text_, not the description _View_. The description view is much wider than the text, because it is `match_parent` but should probably be `wrap_content`.
I don't think this looks right, does it?
![device-2020-08-05-140134](https://user-images.githubusercontent.com/1682214/89448487-b47bde80-d725-11ea-9aff-b05d89d5d06c.png)